sdk/lib/3rdparty/freetype/src/base/ftdebug.c

```diff
@@ -61,10 +62,12 @@
     vfprintf( stderr, fmt, ap );
     va_end( ap );
   }
+#endif
 
 
   /* documentation is in ftdebug.h */
 
+#ifndef __REACTOS__ /* Use DbgPrint instead of this function */
```

Comment:
FT_Panic would be more like a: `DbgPrint + KeBugCheck` instead.
I also wonder whether the vfprintf routine could be aliased to vDbgPrint or EngDebugPrint, with suitable parameters, and the exit() function be aliased to KeBugCheck (actually, EngBugCheckEx if the code needs to import only from win32k), again with suitable parameters.